pkgs/servers/code-server/default.nix

```diff
@@ -57,14 +57,15 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "code-server";
-  version = "4.12.0";
+  version = "4.13.0";
+  commit = "2798322b03e7f446f59c5142215c11711ed7a427";
```

Comment:
Since `commit` is not an attribute of `mkDerivation`, I would move it inside the `let/in` clause up there. WDYT ?

Reply:
Ahh that makes sense!

Reply:
Done.